.gitmodules

```diff
@@ -63,7 +63,7 @@
 	ignore = dirty
 [submodule "submodules/externals/openh264"]
 	path = submodules/externals/openh264
-	url = https://github.com/cisco/openh264
+	url = git://github.com/VTCSecureLLC/openh264.git
```

Comment:
I want to say this line should be:

    url = git://github.com:VTCSecureLLC/openh264.git